src/server/common/helpers/auth/refresh-token.js

```diff
@@ -29,7 +32,6 @@ export async function refreshTokenIfExpired(refreshToken, request, h) {
     try {
       const refreshTokenResponse = await refreshToken(userSession?.refreshToken)
       await refreshUserSession(request, refreshTokenResponse)
-      return h.continue
```

Comment:
Is this functionality for `refreshTokenIfExpired` working as expected? This work is to refresh the token if it has expired and this is done in a `onPreAuth` extension. The uses cases are going to a new page or a page refresh. Or an Ajax call on a page. The reason the Template toolkit was passed through (`h`) was to use `h.continue` to signal to Hapi to continue with the original request if the processing of the refresh token works as expected.

It looks like all this has been removed. Which makes me wonder if its working ok?